pallets/proxy/src/lib.rs

```diff
@@ -546,6 +553,17 @@ pub mod pallet {
             proxy_type: T::ProxyType,
             disambiguation_index: u16,
         },
+        /// A pure proxy was killed by its spawner.
+        PureKilled {
+            // The pure proxy account that was destroyed.
+            pure: T::AccountId,
+            // The account that created the pure proxy.
+            spawner: T::AccountId,
+            // The proxy type of the pure proxy that was destroyed.
+            proxy_type: T::ProxyType,
+            // The index originally passed to `create_pure` when this pure proxy was created.
+            disambiguation_index: u16,
+        },
```

Comment:
The event should be added at the last index, after the existing events or it will mess with SCALE decoding.